src/app/onion/learning-object-builder/builder-store.service.ts

```diff
@@ -825,6 +825,23 @@ export class BuilderStore {
       });
   }
 
+  /**
+   * Public function to remove empty outcomes from a learning object
+   */
+  public async removeEmptyOutcomes() {
+    // Get most up-to-date values for current learning object
+    await this.fetch(this.learningObject.id);
```

Comment:
Where does the learning object id that is fetched go when this request is completed?